client/js/lounge.js

```diff
@@ -79,18 +79,45 @@ $(function() {
 		if (target.hasClass("user")) {
 			output = templates.contextmenu_item({
 				class: "user",
+				action: "whois",
 				text: target.text(),
 				data: target.data("name"),
 			});
+			output += templates.contextmenu_divider();
+			output += templates.contextmenu_item({
+				class: "action-whois",
+				action: "whois",
+				text: "Whois",
+				data: target.data("name"),
+			});
+			output += templates.contextmenu_item({
+				class: "action-query",
+				action: "query",
+				text: "Conversation",
```

Comment:
Let's do "Send a direct message".

Reply:
That is misleading, you are effectively starting a conversation which is different from direct message functionality imho.

Reply:
What about "Direct messages" then (like Slack)? "Conversation" kind of means everything and anything is my issue.

Reply:
How about "private conversation"?

Reply:
"Conversation" still carries this confusing meaning. I prefer "messages" over "conversation". "Private" or "Direct" work, though "private" kinda gives this vibe or privacy, i.e. OTR or something, which we don't do, so I prefer "Direct", but I'm fine with either.

Reply:
What's the problem with Conversation?

Reply:
Still curious about this as well, for all intends and purposes you are starting a conversation as it does open the query window. Direct message more or less implies a fire and forget sort of thing more like mail or private messaging on forums. 

I don't care too strongly either way, matter of fact already changed it but am still curious.

Reply:
It was 2am so maybe I wasn't clear, sorry :)
Issue with this is that "Conversation" is too broad. Actually, let me give you what I expected: when I first ran your PR, I had not looked at the code yet, I wanted to see the look-and-feel without a bias first. When I saw "Conversation", seeing that there was also "Whois" to open a window (with whois information, granted), I was totally expecting that it would bring the nick to the current conversation by auto-completing its name (it's not such a far-fetched idea, it had been brought up by a few people in the past).
I was miles away expecting it would open a new window without the whois :man_shrugging: